EFI/OC/config.plist

```diff
@@ -200,18 +208,18 @@
 	<dict>
 		<key>Add</key>
 		<dict>
+			<key>PciRoot(0x0)/Pci(0x1,0x0)/Pci(0x0,0x0)</key>
```

Comment:
I had to change these lines to boot to `big sur`

``` diff
--- <key>PciRoot(0x0)/Pci(0x1,0x0)/Pci(0x0,0x0)/Pci(0x0,0x0)/Pci(0x0,0x0)</key>
+++ <key>PciRoot(0x0)/Pci(0x1,0x0)/Pci(0x0,0x0)</key>
```

These Changes are added for testing DRM.
https://github.com/risyomei/Hackintosh-ASRock-Z490M-ITX-ac/commit/bd91b3f57c7fe5b43933111f854fc85503e41e9f